microsoft-365/commerce/licenses/manage-license-requests.md

```diff
@@ -22,7 +22,7 @@ description: "Learn how to review and approve or deny license requests from user
 ms.date: 04/22/2022
 ---
 
-# Manage license requests
+# Manage self-service license requests
```

Comment:
@lhcole613 - Add product name for SEO?

Reply:
@SKjerland To be honest, I wasn't sure what product name to use with this one and where. Should it be: Manage Microsoft 365 self-service license requests

Reply:
@lhcole613 - The product name placement looks good, but since this is Cern's area, we should probably ask him to confirm the product. @cmcatee-MSFT - Are we talking about Microsoft 365 licenses here?